core/src/main/java/io/confluent/rest/SslFactory.java

```diff
@@ -50,12 +50,12 @@ public static SslContextFactory createSslContextFactory(SslConfig sslConfig) {
           FileWatcher.onFileChange(watchLocation, () -> {
                 // Need to reset the key store path for symbolic link case
                 sslContextFactory.setKeyStorePath(sslConfig.getKeyStorePath());
-                sslContextFactory.reload(scf -> log.info("Reloaded SSL cert"));
+                sslContextFactory.reload(scf -> log.info("SSL cert auto reload begun"));
               }
```

Comment:
Can we add a log after SSL reload completes

Reply:
Couldn't figure out how to do it.

Reply:
Can't we add a log after `sslContextFactory.reload` line

Reply:
Is it a properly synchronous call with all side-effects guaranteed to have been completed by the time it returns? Why does the reload method take a `Consumer` if it's actually no use at all? I will add a log line. I doubt it is really that informative in practice.